doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -6,6 +6,8 @@
 - Unused setting `general.language` has been deprecated.
 #### [Listing]
 - The methods `setOrder()` and `setOrderKey()` throw an `InvalidArgumentException` if the parameters are invalid now.
+#### [Html to Image]
+- [Gotenberg] Bumped the lowest requirement of `gotenberg-php` from `^2.0` to `^2.4` to add support of passing screenshot size
```

Comment:
does it sounds clear that the minimum `^1.1` is still supported?

Reply:
Agree that it's potentially a little bit strange, but I would say it's fine. Some kind of convoluted sentence which describes the reason for the ^1.1 support wouldn't make it better, so let's write it like this.